README.md

```diff
@@ -19,7 +19,7 @@ Use a `.env` at root of the repository to set values for the environment variabl
 | DB_USERNAME            |    Y     |          -          | The database username                                                                        |
 | DB_PASSWORD            |    Y     |          -          | The database password                                                                        |
 | IDENTITY_SERVICE_HOST  |    Y     |          -          | Hostname of the `dscp-identity-service`                                                      |
-| IDENTITY_SERVICE_PORT  |    N     |       `3000`        | Port of the `dscp-identity-service`                                                          |
+| IDENTITY_SERVICE_PORT  |    N     |       `3002`        | Port of the `dscp-identity-service`                                                          |
```

Comment:
The default here is currently correct based on the `default` (not `devDefault`) in [`env.ts` ](https://github.com/digicatapult/dscp-hyproof-api/blob/5fa4b7ab2636dc934b66f0467a26381668505443/src/env.ts#L20)

Reply:
Reverted